src/Hooks/useClientVersion.ts

```diff
@@ -0,0 +1,3 @@
+export const useClientVersion = () => {
+    return PROJECT_VERSION;
+  };
```

Comment:
file doesn't have a newline at the end. do you have the editorconfig plugin? https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

it will newlines etc. automatically